app/packages/components/src/components/Selection/SearchBox.tsx

```diff
@@ -51,6 +51,7 @@ export const SearchBox = ({
   } = theme;
   return (
     <Box
+      data-cy="search-container"
```

Comment:
Could we also prefix `data-cy`s with a string that describes the context within which it exists? For example, here, it could be `selection-search-container`?